docs/features/network-security-controls/network-policy.md

```diff
@@ -33,7 +149,9 @@ spec:
   - Egress
 ```
 
-If only ingress traffic to all pods in a namespace needs to be blocked the following can be used 
+#### **Ingress default-deny**
+
+When the podSelector field is wildcarded, the `policyTypes` is set to `Ingress`, and no ingress rules are specified, all ingress traffic to the namespace is blocked.
```

Comment:
I am not sure what "podSelector field is wildcarded" means, can you please give an example?